commons/connector/health/health_height.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "namespace": "org.radarcns.connector.health",
+  "name": "HKQuantityTypeIdentifierHeight",
```

Comment:
I think this can be combined with other body measurements as one schema. Maybe called `HKQuantityTypeIdentifierBodyMeasurements`. This can be combined with height, body mass, body mass index, lean body mass, body fat percentage, and weight circumference. As categorised here: https://developer.apple.com/documentation/healthkit/